lib/opensusebasetest.pm

```diff
@@ -29,6 +30,53 @@ sub export_logs {
     select_console 'root-console';
     save_screenshot;
 
+    # check if wicked is installed and turn on debug, get all needed wicked logs if network interface is down
+    if (!assert_script_run 'rpm -q wicked') {
```

Comment:
this is an interesting usage since assert_script_run() always return undef if it's on success(no die), assert_script_run() is not intentional use of conditional statement. but *maybe* it's a good point here, IMO we can make it return 1 or ```$ret``` in https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L631 , it should/will not break anything else(maybe t/03-testapi). what do you think everybody?